Sources/ProjectDescription/Target.swift

```diff
@@ -31,7 +31,7 @@ public class Target: Codable {
     let sources: String
```

Comment:
What about turning this one into an array as well?

Reply:
I changed `sources` to [String] as well.